controlplane/kubeadm/internal/controllers/controller_test.go

```diff
@@ -918,7 +918,7 @@ kubernetesVersion: metav1.16.1`,
 	g.Expect(s.Data).NotTo(BeEmpty())
 	g.Expect(s.Labels).To(Equal(expectedLabels))
 
-	k, err := kubeconfig.FromSecret(ctx, fakeClient, util.ObjectKey(cluster))
+	k, err := kubeconfig.FromSecret(ctx, fakeClient, util.ObjectKey(cluster), true)
```

Comment:
Again, i think this should be `false` to preserve the existing behaviour. The only place where we should get the user kubeconfig by default is from the clusterctl command.

Reply:
+1 for keeping existing behaviour